src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: (1, 200px);
+
+  gap: 30px 48px;
+  font-family: $font-family-roboto;
+
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    transition-duration: 300ms;
+
+    &_image {
+      display: block;
+      margin: 0 auto 40px;
+      width: 160px;
+      height: 134px;
+    }
+
+    &_name {
+      display: block;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      width: 166px;
+      color: $main-color;
+      margin-bottom: 4px;
+      margin-top: 0;
```

Comment:
The width of '.products__card_name' is set to 166px, which might not be flexible if the card layout changes. Consider using 'max-width' or a percentage value to maintain flexibility.